src/templates/nav-button.jsx

```diff
@@ -18,7 +26,10 @@ export const NavButton = React.createClass({
                 disabled={!this.props.isEnabled}
                 href="#"
                 onClick={this.props.isEnabled ? this.navigate : null}
-            />
+            >
+              <Gridicon icon="arrow-down" size={18} />
```

Comment:
It looks like we do have an `arrow-up` gridicon as well. I'd prefer if we used it directly since a `transform` may make the icon look a bit blurry.

We have a few options here, we can toggle based off the className, or add an explicit prop. I prefer the latter.

So this might look like
```jsxt
  <NavButton
       className="wpnc__prev"
       iconName="arrow-up"
       //...
  />
  <NavButton
       className="wpnc__next"
       iconName="arrow-down"
       //...
  />
```

Where in this component we do:
```jsx
<Gridicon icon={ this.props.iconName } size={ 18 } />
```